### PR TITLE
Feature/m0078 ddg 553 mej32 block diagram

### PR DESCRIPTION
**Pull request by frivieremicroej**
Add final Core Engine block diagram and refresh the whole Core Engine page with latest glossary terms.

### CODE REVIEW

File: VEEPortingGuide/coreEngine.rst [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
+It executes at runtime the Application code.
 
+Block Diagram
+=============
 
-Functional Description
-======================
+.. figure:: images/mjvm_block-diagram.png
+   :alt: Core Engine Block Diagram
+   :align: center
```
Comment (jerome-leroux) [inline VEEPortingGuide/coreEngine.rst:16]: The image is not clickable to open it full screen.

File: VEEPortingGuide/images/mjvm_block-diagram.png
Comment (jerome-leroux) [inline VEEPortingGuide/images/mjvm_block-diagram.png:1]: 32bits -> 32-bit
64bits -> 64-bit
16bits -> 16-bit

Comment (jerome-leroux) [inline VEEPortingGuide/images/mjvm_block-diagram.png:1]: In "Advanced Profiling", add "Exceptions"

File: VEEPortingGuide/coreEngine.rst [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
+
+   Core Engine Block Diagram
+
+Link Flow
+=========
 
-:ref:`The following diagram <fig_mjvm_flow2>` shows the overall process. The first two
-steps are performed within the MicroEJ Workbench. The remaining steps
-are performed within the C IDE.
+:ref:`The following diagram <fig_mjvm_flow2>` shows the overall process.
```
Comment (jerome-leroux) [inline VEEPortingGuide/coreEngine.rst:23]:
```suggestion
:ref:`The following diagram <fig_mjvm_flow2>` shows the overall build flow. 
```

**Comment by jerome-leroux** [comment]
Changes to be merge with what have been done for Architecture 8.1. See https://github.com/MicroEJ/docs/compare/master...architecture_8_1#diff-52e36b4feaff961f710ef9ed6dccea00ac5cd9119f8ef16c6167cba9b1a8935d

File: VEEPortingGuide/coreEngine.rst [hunk 3]
```diff
@@ -1,38 +1,46 @@
 .. _core_engine:
```
Comment (jerome-leroux) [inline VEEPortingGuide/coreEngine.rst:1]: Changes to be merged with what have been done for Architecture 8.1: https://github.com/MicroEJ/docs/compare/master...architecture_8_1#diff-52e36b4feaff961f710ef9ed6dccea00ac5cd9119f8ef16c6167cba9b1a8935d

File: VEEPortingGuide/coreEngine.rst [hunk 4]
```diff
@@ -155,17 +166,17 @@ to disable the round-robin scheduling.
...
+into a sleep state. ``LLMJVM_IMPL_wakeupVM`` is called
+to wake up the Core Engine RTOS task. When the Core Engine RTOS task really starts to
 execute again, it calls the ``LLMJVM_IMPL_ackWakeup`` function to
 acknowledge the restart of its activity.
 
 Time
 ----
 
-The platform defines two times:
+The Core Engine defines two times:
```
Comment (jerome-leroux) [inline VEEPortingGuide/coreEngine.rst:179]:
```suggestion
The Core Engine defines two different times:
```

**Review by jerome-leroux — CHANGES_REQUESTED** [review]

File: VEEPortingGuide/coreEngine.rst (hunk 3 above)
Comment (frivieremicroej) [reply VEEPortingGuide/coreEngine.rst:1]: OK, I suggest to first merge this branch asap, then I will rebase the Architecture 8.1 branch on top of it

**Review by jerome-leroux — APPROVED** [review]

**Pull request merged**